x-pack/plugins/rollup/public/crud_app/sections/job_list/job_list.js

```diff
@@ -0,0 +1,246 @@
+          title={title}
+          color="warning"
+          iconType="help"
+        >
+          <FormattedMessage
+            id="xpack.rollupJobs.jobList.noPermissionText"
+            defaultMessage="You do not have permission to view or add rollup jobs."
+          />
+        </EuiCallOut>
+      </Fragment>
+    );
+  }
+  renderEmpty() {
+    return (
+      <EuiEmptyPrompt
+        iconType="managementApp"
+        title={(
+          <h1>
+            <FormattedMessage
+              id="xpack.rollupJobs.jobList.emptyPromptTitle"
+              defaultMessage="Create your first rollup job"
+            />
+          </h1>
+        )}
+        body={
+          <Fragment>
+            <p>
+              <FormattedMessage
+                id="xpack.rollupJobs.jobList.emptyPromptDescription"
+                defaultMessage={`
+                  Rollup jobs summarize and store your data in a format that's compressed, yet aggregatable.
+                `}
+              />
+            </p>
+          </Fragment>
+        }
+        actions={
+          <EuiButton
+            {...getRouterLinkProps(`${CRUD_APP_BASE_PATH}/create`)}
+            fill
+            iconType="plusInCircle"
+          >
+            <FormattedMessage
+              id="xpack.rollupJobs.jobList.emptyPrompt.createButtonLabel"
+              defaultMessage="Create rollup job"
+            />
+          </EuiButton>
+        }
+      />
+    );
+  }
+
+  renderList() {
+    const { isLoading } = this.props;
+
+    let table;
+
+    if (isLoading) {
+      table = (
+        <EuiFlexGroup
+          justifyContent="flexStart"
+          alignItems="center"
+          gutterSize="s"
+        >
+          <EuiFlexItem grow={false}>
+            <EuiLoadingSpinner size="m" />
+          </EuiFlexItem>
+
+          <EuiFlexItem grow={false}>
+            <EuiText>
+              <EuiTextColor color="subdued">
+                <FormattedMessage
+                  id="xpack.rollupJobs.jobList.loadingTitle"
+                  defaultMessage="Loading rollup jobs..."
+                />
+              </EuiTextColor>
+            </EuiText>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      );
+    } else {
+      table = <JobTable />;
+    }
+
+    return (
+      <Fragment>
+        <EuiPageContentHeader>
+          {this.getHeaderSection()}
+
+          <EuiPageContentHeaderSection>
+            <EuiButton fill {...getRouterLinkProps(`${CRUD_APP_BASE_PATH}/create`)}>
+              <FormattedMessage
+                id="xpack.rollupJobs.jobList.createButtonLabel"
+                defaultMessage="Create rollup job"
+              />
+            </EuiButton>
+          </EuiPageContentHeaderSection>
+        </EuiPageContentHeader>
+
+        {table}
+
+        <DetailPanel />
+      </Fragment>
+    );
+  }
+
+  render() {
+    const { isLoading, jobs, jobLoadError } = this.props;
+
+    let content;
+    if (jobLoadError && jobLoadError.status === 403) {
```

Comment:
If we have a jobLoadError and it's not because of a permission error, we should probably do something besides load an empty screen, right?

Reply:
https://github.com/elastic/kibana/issues/24777